source/documentors/how-tos/update_a_doc_via_github.rst

```diff
@@ -1,32 +1,34 @@
 Update A Doc via GitHub
-###############################
+#######################
 
 This how-to will take you through the process of submitting an update to a page
-that you are looking at within the Open edX documnetation.
+that you are looking at within the Open edX documentation.
 
 Prerequisites
 *************
 
-* You're on the published documentation page at some address like
-  ``docs.openedx.org/...`` and you want to make changes to it.
+#. You're on the published documentation page at some address like ``docs.openedx.org/...`` and you want to make changes to it.
+
+#. You have a GitHub Account
 
-* You have a GitHub Account
    * If not `sign up here`_.
 
-.. _sign up here: https://github.com/signup
+#. In the GitHub UI, make any edits you need to make.
```

Comment:
Nit: Numeric pre-reqs feels weird to me since they're not inter-related steps but not a huge deal if you like it this way.